scripts/isolate_tests.py

```diff
@@ -96,7 +74,7 @@ def write_cases(f, tests):
 
 def extract_and_write(f, path):
     if docs:
-        cases = extract_docs_cases(path)
+        cases = extract_docs_cases(path, ".. code-block:: solidity")
```

Comment:
You are concatenating this with a regex so you need to escape the dots or they will match any character.
```suggestion
        cases = extract_docs_cases(path, r'\.\. code-block:: solidity')
```

Reply:
Actually, no I am not. This is only used in pure str-compare matching.

Reply:
Ah, you're probably right. I must have mistaken this `codeStart` in `r'^\s{4}' + codeStart` for this parameter.